src/ui/components/molecules/BannerNavbar/bannerNavbar.style.ts

```diff
@@ -0,0 +1,82 @@
+import type { Theme } from '@mui/material/styles';
+import type { CSSProperties } from 'react';
+
+export const bannerNavbarStyles = (theme: Theme) => ({
+  appBar: {
+    background: '#0793BF',
```

Comment:
dont hardcode the color directly please use the theme provider, something like 
```
background: theme.color.background
```
Please review the material ui documentation i not sure about that properties